secrets/lib/secrets.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Secrets do
+  def secret_add(secret) do
+    fn add -> add + secret end
+     
+  end
+
+  def secret_subtract(secret) do
+    fn sub -> sub - secret end
+  end
+
+  def secret_multiply(secret) do
+     fn mult -> mult * secret end
+  end
+
+  def secret_divide(secret) do
+     fn div -> trunc(div/secret) end
```

Comment:
This is recommended `div / secret` rather than `div/secret`. Remember this as a thumb rule whenever you are performing mathematical operations